core/controllers/topic_viewer.py

```diff
@@ -140,6 +131,6 @@ def get(self, topic_name):
             'subtopics': subtopics,
             'degrees_of_mastery': degrees_of_mastery,
             'skill_descriptions': skill_descriptions,
-            'train_tab_should_be_displayed': train_tab_should_be_displayed
+            'practice_tab_is_displayed': topic.practice_tab_is_displayed
```

Comment:
Doesn't seem to be related to the title of this PR? (Maybe update the title?)

Reply:
Ah yes, I had added it in the description but not in the title initially. But I've updated the title now, thanks!